torchbenchmark/models/BERT_pytorch/bert_pytorch/model/embedding/position.py

```diff
@@ -10,7 +10,7 @@ def __init__(self, d_model, max_len=512):
 
         # Compute the positional encodings once in log space.
         pe = torch.zeros(max_len, d_model).float()
-        pe.require_grad = False
+        pe.requires_grad = False
```

Comment:
Could you please add `https://github.com/codertimo/BERT-pytorch/pull/104` here so that we can track the upstream code change?

Reply:
done